[DOC-13540]: Documented new log upload notification feature and added usage instructions.

RayOffiah opened the pull request:
Added documentation for the new log upload notification feature, including usage guidelines. 

Preview URL:
https://preview.docs-test.couchbase.com/docs-server-DOC-13540-Document-New-Log-Transfer-Opt-In-Feature-with-Customer-Notifications

You will need the [Docs Team credentials](https://confluence.issues.couchbase.com/wiki/spaces/DOCS/pages/1971224949/Docs+Team+demo+site+passwords) on Confluence.

RayOffiah committed:
[DOC-13540]: Documented new log upload notification feature and added usage instructions.

Signed-off-by: Ray Offiah <ray.offiah@couchbase.com>

gsauere commented on modules/manage/pages/manage-logging/manage-logging.adoc:
```diff
@@ -94,6 +94,32 @@ Otherwise, if the checkbox _is_ checked, the upload specifications are submitted
+
+Couchbase Server offers the facility to send an alert on receipt of an uploaded log file.
+This function is offered on an opt-in basis.
+Contact Couchbase Support for more information.
```
Contact your Account Manager or Couchbase Support for more information.

gsauere commented on modules/manage/pages/manage-logging/manage-logging.adoc:
```diff
@@ -94,6 +94,32 @@ Otherwise, if the checkbox _is_ checked, the upload specifications are submitted
+
+image::manage-logging/logUploadForAlert.png[, 500,align=left]
+
+
```
It will be nice to include an image to show how customers will see the updates on the support ticket.

RayOffiah replied:
Okay, I'll sort something out.

RayOffiah committed:
[DOC-13540]: Documented new log upload notification feature and added usage instructions.

Added documentation for log upload acknowledgment with accompanying images. Updated instructions for contacting support and uploading logs.

Signed-off-by: Ray Offiah <ray.offiah@couchbase.com>

JackBakes commented on modules/manage/pages/manage-logging/manage-logging.adoc:
```diff
@@ -94,6 +94,40 @@ Otherwise, if the checkbox _is_ checked, the upload specifications are submitted
+[.status]#Couchbase Server Enterprise#
+
+
+Couchbase Server offers the facility to send an alert on receipt of an uploaded log file.
```
Nitpick, but it's not Couchbase Server (the product) offering this facility, it's Couchbase the organization, or specifically the Support Team.

I think I'd suggest the following:

```suggestion
Couchbase Customer Support offers the facility to send you an automatic notification on receipt of any log file uploaded for a support case.
```

RayOffiah committed:
[DOC-13540]: Documented new log upload notification feature and added usage instructions.

Clarified new section.

Signed-off-by: Ray Offiah <ray.offiah@couchbase.com>
